### PR TITLE
Fix styling issue with ORA staff tools

### PR DESCRIPTION
**Pull request by andy-armstrong**
## [TNL-3845](https://openedx.atlassian.net/browse/TNL-3845)

Applied @frrrances's suggested fix for the clash of XBlock and ORA stylings in the release candidate.

### Sandbox
- [x] https://andy-armstrong.sandbox.edx.org

### Testing
- [x] Manual verification locally and on the sandbox
  - https://andy-armstrong.sandbox.edx.org/courses/course-v1:edx+ORA203+course/courseware/a4dfec19cf9b4a6fb5b18be6ccd9cecc/338a4affb58a45459629e0566291381e/

### Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [x] Code review: @cahrens 
- [x] UX review: @frrrances 

FYI: @robrap 

### Post-review
- [ ] Squash commits

### CODE REVIEW

**Comment by andy-armstrong** [comment]
Note: I still need to cut an ORA release from this commit and then provide a patch to the RC.

File: openassessment/xblock/static/sass/oa/elements/_staff.scss [hunk 1]
```diff
@@ -91,7 +91,7 @@
   .staff-info__status__table, .staff-info__classifierset__table {
     @extend %copy-3;
     @include border-radius(($baseline-v/10));
-    color: $copy-staff-color;
```
Comment (cahrens) [inline openassessment/xblock/static/sass/oa/elements/_staff.scss]: I this color just unnecessary because it inherits it?

**Comment by cahrens** [comment]
:+1: Thanks for following up on this and fixing it, @andy-armstrong.

File: openassessment/xblock/static/sass/oa/elements/_staff.scss (hunk 1 above)
Comment (andy-armstrong) [inline openassessment/xblock/static/sass/oa/elements/_staff.scss]: Hmmm, that was an accident. However, the styling seems fine without it. :-)

**Comment by frrrances** [comment]
:+1:

**Pull request merged**